src/Stryker.Core/Stryker.Core/Mutants/MutantPlacer.cs

```diff
@@ -52,13 +52,25 @@ private static SyntaxNode RemoveByIfStatement(IfStatementSyntax ifStatement)
         public static ParenthesizedExpressionSyntax PlaceWithConditionalExpression(ExpressionSyntax original, ExpressionSyntax mutated, int mutantId)
         {
             // place the mutated statement inside the if statement
-            return SyntaxFactory.ParenthesizedExpression(
+            var result = SyntaxFactory.ParenthesizedExpression(
                 SyntaxFactory.ConditionalExpression(
                 condition: GetBinaryExpression(mutantId),
                 whenTrue: mutated,
                 whenFalse: original))
                 // Mark this node as a MutationConditional node. Store the MutantId in the annotation to retrace the mutant later
                 .WithAdditionalAnnotations(new SyntaxAnnotation(MutationConditional, mutantId.ToString()));
+            // provides the new location of the original node
```

Comment:
What does this comment mean? It doesn't add much clarity for me. Also the variable declaration doesn't seem necessary here.